### PR TITLE
Update and fix Spanish locale

### PR DESCRIPTION
**Pull request by EdgarLux**
I noticed file is outdated (old template) and has reverted many new translations lines. Fix commit cf011f5.

### CODE REVIEW

**Comment by TurboGit** [comment]
Thanks!

File: po/es.po [hunk 1]
```diff
@@ -6099,181 +6199,182 @@ msgstr "controles"
...
+#: ../src/iop/colorbalance.c:935 ../src/iop/colorbalance.c:2610
 msgid "neutralize colors"
-msgstr "neutralizar los colores"
+msgstr "neutraliza los colores"
 
-#: ../src/iop/colorbalance.c:2324
+#: ../src/iop/colorbalance.c:2316
 msgid "lift, gamma, gain (ProPhotoRGB)"
-msgstr "realce, gamma, ganancia (ProPhotoRGB)"
+msgstr "elevación, gamma, ganancia (ProPhotoRGB)"
```
Comment (agusmba) [inline po/es.po]: Even though I'm Spanish I have no idea about photography terminology, so I'm not sure about the "lift" -> "elevación" translation, which sounds like a direct translation, but in my mind, "realce" sounds better

I'm not asking for a change here (and everywhere else this appears) but for confirmation that this is indeed the right translation.

File: po/es.po [hunk 2]
```diff
@@ -6939,27 +7042,27 @@ msgstr "deshabilitado"
...
 msgid "green channels matching method"
-msgstr "método de proceso del canal verde"
+msgstr "método de emparejamiento para el canal verde"
 
 #: ../src/iop/demosaic.c:5115
 msgid ""
 "demosaicing\n"
 "only needed for raw images."
 msgstr ""
 "demosaico\n"
```
Comment (agusmba) [inline po/es.po]: this has been translated differently elsewhere: "interpolación cromática"?

File: po/es.po [hunk 3]
```diff
@@ -6972,17 +7075,17 @@ msgstr "croma (uso en 1ª instancia)"
 
 #: ../src/iop/denoiseprofile.c:340
 msgid "luma (use on 2nd instance)"
-msgstr "luminosidad (usar en 2a instancia)"
+msgstr "luma (uso en 2º instancia)"
```
Comment (agusmba) [inline po/es.po]: 2ª instancia

File: po/es.po [hunk 4]
```diff
@@ -7002,32 +7105,32 @@ msgstr "tamaño del cuadro"
 
 #: ../src/iop/denoiseprofile.c:2788
 msgid "search radius"
-msgstr "radio de búsqueda"
+msgstr "buscar el radio"
 
 #: ../src/iop/denoiseprofile.c:2790
 msgid "scattering (coarse-grain noise)"
-msgstr "dispersión (ruido de grano grueso)"
+msgstr "esparcir (ruido con granularidad gruesa)"
```
Comment (agusmba) [inline po/es.po]: dispersión sounds better

File: po/es.po [hunk 5]
```diff
@@ -7039,22 +7142,22 @@ msgid ""
...
+"grandes pueden ser muy lentos. se le ha advertido"
 
 #: ../src/iop/denoiseprofile.c:2801
 msgid ""
 "scattering of the neighbourhood to search patches in. increase for better "
 "coarse-grain noise reduction. does not affect execution time."
 msgstr ""
-"dispersión del mas cercano para buscar parches en. incremento para una mejor "
-"reducción del ruido de grano grueso. no afecta al tiempo de ejecución."
+"dispersión en la proximidad para buscar coincidencias. aumente para una "
```
Comment (agusmba) [inline po/es.po]: ... Aumente ...

File: po/es.po [hunk 6]
```diff
@@ -7286,249 +7391,253 @@ msgstr "16 EV (HDR)"
...
-#: ../src/iop/filmic.c:1692
+#: ../src/iop/filmic.c:1682
 msgid "white relative exposure"
-msgstr "exposición relativa del blanco"
+msgstr "exposición blanca relativa"
 
-#: ../src/iop/filmic.c:1695
+# número de pasos entre el gris medio y el blanco puro.
+# esta es una lectura que un fotómetro te daría en la escena.
+# ajustar para evitar el recorte de altas luces
```
Comment (agusmba) [inline po/es.po]: I'm guessing these commented out lines shouldn't be here?

File: po/es.po [hunk 7]
```diff
@@ -7286,249 +7391,253 @@ msgstr "16 EV (HDR)"
...
 "increase to get more contrast.\n"
 "decrease to recover more details in low-lights."
 msgstr ""
-"número de stops entre gris medio y negro puro.\n"
-"esta es una lectura que un exposimetro le daría en la escena.\n"
-"incrementar para obtener más contraste.\n"
-"disminuir para recuperar más detalles en condiciones de poca luz."
+"número de pasos entre el gris medio y el negro puro.\n"
+"esta es una lectura que un fotómetro te daría en la escena.\n"
+"para obtener más contraste.\n"
```
Comment (agusmba) [inline po/es.po]: why remove the verb? "incrementar" seems necessary to understand this

File: po/es.po [hunk 8]
```diff
@@ -7286,249 +7391,253 @@ msgstr "16 EV (HDR)"
...
+"para recuperar más detalles en condiciones de poca luz."
```
Comment (agusmba) [inline po/es.po]: why remove the verb? "disminuir/decrementar" seems necessary to understand this

File: po/es.po [hunk 9]
```diff
@@ -7286,249 +7391,253 @@ msgstr "16 EV (HDR)"
...
+"esto encajará con el rango de luminancia dentro de los límites del "
+"histograma.\n"
+"funciona mejor para paisajes y fotos uniformemente iluminadas\n"
+"pero falla en las clave alta y clave baja."
 
-#: ../src/iop/filmic.c:1738
+#: ../src/iop/filmic.c:1728
 msgid "filmic S curve"
-msgstr "curva S filmica"
+msgstr "curva S de película"
```
Comment (agusmba) [inline po/es.po]: Depends on how the "filmic" module name is translated. "película" seems off

File: po/es.po [hunk 10]
```diff
@@ -8074,19 +8185,19 @@ msgstr "estereográfico"
 
 #: ../src/iop/lens.c:2282
 msgid "equisolid angle"
-msgstr "ángulo equisólido"
+msgstr "ángulo equisolid"
```
Comment (agusmba) [inline po/es.po]: why?

File: po/es.po [hunk 11]
```diff
@@ -8509,80 +8625,80 @@ msgstr "rango de sombras"
 
 #: ../src/iop/profile_gamma.c:153
 msgid "16 EV dynamic range (generic)"
-msgstr "16 EV rango dinámico (genérico)"
+msgstr "16EV  Rango dinámico  (genérico)"
```
Comment (agusmba) [inline po/es.po]: spacing?

**Comment by EdgarLux** [comment]
I really appreciate that you made these observations. I will review them. It seems that there was a parallel correction and there are others that I will keep checking. @agusmba

File: po/es.po [hunk 12]
```diff
@@ -136,101 +140,114 @@ msgstr ""
...
 msgstr ""
-"esta opción alterna el comportamiento del shift clic en el modo de mesa de "
+"esta opción alterna el comportamiento del máyus+clic en el modo de mesa de "
 "luz"
 
-#: ../build/src/preferences_gen.h:2111
+#: ../build/src/preferences_gen.h:2231
 msgid "scroll to lighttable modules when expanded/collapsed"
-msgstr "desplazar a los módulos de la mesa de luz al expandirlos/colapsarlos"
+msgstr "desplazar a los módulos de la mesa de luz al expanderlos/colapsarlos"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: It's a typo. Thanks!

File: po/es.po [hunk 13]
```diff
@@ -288,105 +305,119 @@ msgstr ""
...
+#: ../build/src/preferences_gen.h:2473
 msgid "scroll to darkroom modules when expanded/collapsed"
-msgstr "desplazar a los módulos del cuarto oscuro al expandirlos/colapsarlos"
+msgstr "desplazar a los módulos del cuarto oscuro al expanderlos/colapsarlos"
 
-#: ../build/src/preferences_gen.h:2353
+#: ../build/src/preferences_gen.h:2493
 msgid "border around image in darkroom mode"
-msgstr "borde alrededor de la imagen en el modo de cuarto oscuro"
+msgstr "borde al rededor de la imagen en el modo de cuarto oscuro"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: It's a typo. Thanks!

File: po/es.po [hunk 14]
```diff
@@ -288,105 +305,119 @@ msgstr ""
...
 "don't want any border."
 msgstr ""
 "procesa la imagen en el modo de cuarto oscuro con un borde mas pequeño. "
 "ajuste a 0 si no quiere ningún borde."
 
-#: ../build/src/preferences_gen.h:2380
+#: ../build/src/preferences_gen.h:2520
 msgid "map / geolocalisation"
-msgstr "map / geolocalización"
+msgstr "mapa/geolocalización"
```
[the remark this answers is not in the document]
Comment (EdgarLux) [reply po/es.po]: Yes, indeed.

**Pull request merged**